Source/Tools/LinuxEmulation/LinuxSyscalls/ThreadManager.h

```diff
@@ -200,8 +200,11 @@ class ThreadManager final {
     auto CodeInvalidationlk = GuardSignalDeferringSectionWithFallback(CTX->GetCodeInvalidationMutex(), CallingThread);
 
     for (auto& Thread : Threads) {
-      CTX->InvalidateGuestCodeRange(Thread->Thread, Start, Length, callback);
+      CTX->InvalidateGuestCodeRange(Thread->Thread, Start, Length);
```

Comment:
I think we can remove the `callback` parameter from this function's signature now, as it's never used anywhere.

Reply:
Removed.